new/LIT7052Qene.xml

```diff
@@ -6,8 +6,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <fileDesc>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">ኢሀሎ፡ በዓለም፡ ዲበ፡ ሰረገላ፡ እንዘ፡ በዘባኑ፡ ለአድግ፡ ነባቤ፡ ነገር፡ ሠናይ።</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">ʾI-hallo ba-ʿālam diba saragalā ʾǝnza ba-zabānu la-ʾadg nabābe nagar śannāy.</title>
-                <title xml:lang="en" corresp="#t1">There is nothing in the world on the chariot, since the speaker of the good news is on the back of a donkey. ...</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾI-hallo ba-ʿālam diba saragallā ʾǝnza ba-zabānu la-ʾadg nabābe nagar śannāy.</title>
+                <title xml:lang="en" corresp="#t1">In the world, he is not on the chariot, but on the back of the donkey - the speaker of the good things. ...</title>
```

Comment:
I would use comma after donkey (open the possibility of interpretation who the speaker is, He or donkey :) ). I am not fully certain that I fully understand it ()